sdk/clientcore/http-okhttp3/pom.xml

```diff
@@ -85,8 +85,8 @@
     <!-- Test dependencies on core, because we want to run tests inherited from this module using OkHttp -->
     <dependency>
       <groupId>io.clientcore</groupId>
-      <artifactId>core</artifactId>
-      <version>1.0.0-beta.8</version> <!-- {x-version-update;io.clientcore:core;current} -->
+      <artifactId>annotation-processor-test</artifactId>
```

Comment:
I don't know how to feel about this change. This feels to be breaking the separation of Annotation Processor projects from the other ClientCore projects. Which was the whole reasons we added a different CI pipeline for annotation-processor.